package.json

```diff
@@ -119,6 +121,7 @@
     "conditional-wrap": "1.0.2",
     "console-browserify": "1.2.0",
     "constants-browserify": "1.0.0",
+    "currency.js": "2.0.4",
```

Comment:
used for addition of currencies, this is a one js file that we can copy paste in the app if we want